src/qml/controls/OnboardingInfo.qml

```diff
@@ -49,11 +53,40 @@ Control {
             subtextMargin: root.subtextMargin
             subtextSize: root.subtextSize
         }
-        ContinueButton {
-            Layout.alignment: Qt.AlignCenter
-            Layout.topMargin: 40
-            text: root.buttonText
-            onClicked: swipeView.incrementCurrentIndex()
-        }
     }
+    ContinueButton {
+        id: continueButton
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.topMargin: 40
+        anchors.bottomMargin: 60
+        anchors.leftMargin: 20
+        anchors.rightMargin: 20
+        text: root.buttonText
+        onClicked: swipeView.incrementCurrentIndex()
+    }
+
+    state: AppMode.state
+
+    states: [
```

Comment:
An alternative would be to define anchor values for components inside of our theme file, and assign values in the same way we assign the color values; but when the mode changes, the anchor values change. I’ll make a branch implementing this idea and we can compare

Reply:
The gist of the idea is: 

This checking of the AppMode state can just be hidden inside of the Theme file, and the theme passes the values we want to the component

Reply:
I'm not sure if it makes sense to move anchor values outside of the component definition since they are relative values that only make sense in a context that includes all of the components involved.

One thing I also learned while doing this is that Qt doesn't recommend conditional anchors bindings. "Because the evaluation order of bindings is not defined, it is not recommended to change anchors via conditional bindings, as this can lead to the ordering issue described above. In the following example the Rectangle will eventually grow to the full width of its parent, because both left and right anchors will be simultaneously set during binding update." (https://doc.qt.io/qt-6/qtquick-positioning-anchors.html#changing-anchors). That was my primary reason for using states and AnchorChanges which unfortunately requires more boilerplate than just using a tertiary.